NoteFind/Models/Tags.h

```diff
@@ -7,12 +7,13 @@
 
 #import <Parse/Parse.h>
 #import "PFObject.h"
+#import "Note.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface Tags : PFObject <PFSubclassing>
 @property (nonatomic, strong) NSString *title;
-
+@property (nonatomic, strong) NSMutableArray *taggedNotes;
```

Comment:
Lets add a generic type here for `NSMutableArray<>`, this allows the compiler to do a better job of knowing what types of values should be contained and returned when using this collection

https://useyourloaf.com/blog/using-objective-c-lightweight-generics/

Reply:
resolved!